### ISSUE TITLE
Data Template - April 22

### ISSUE THREAD

**Issue by neil-ices-dk** [ices-tools-dev/cetaceans#257]
@cmspinto to test the idea of an additional header record (non-mandatory) which would allow multiple surveys to be submitted as one file (rather than separate files for each survey). This would mainly affect the backend system - however needs to investigate a little further.

**Comment by cmspinto** [comment]
This is concluded and the new template is now finished. 
It was possible to allow the submissions of multiple surveys in one file and still be compatible with the previous format. 
So the inclusion of a root header when the file has more than a survey is optional.

We should consider in the next revision of the format to pass some of the fields in the Identifier record to the file header and make the file header mandatory record.

**Comment by NikiClear** [comment]
Hi Carlos,

This is great news!!

I've been testing with a re-upload of SCANS, and the process worked perfectly. However,

While trying to re-submit the SCANS data, I'm getting an error message (similar to what I was getting before). The data seems to have passed all verification, except the overwrite warning but I can't seem to proceed to upload. Here is a screen grab of what I get when I click 'Import cetaceans data to the database and overwrite the file in the database' button.

[image]<https://user-images.githubusercontent.com/87769388/168302751-e6b90cd1-1df3-48d2-a77f-a65ced7b7e1d.png>


I think I had this error before. I was wondering if you could help me, and test to make sure this won't happen to others who try to upload?
(I've also added this issue to GitHub - https://github.com/ices-tools-dev/cetaceans/issues/257)

Thanks,
Niki
________________________________

**Comment by cmspinto** [comment]
Hi Niki,

Can you send me the file or what is the SessionID?

**Comment by NikiClear** [comment]
Hi,

Session ID is 1517<https://cetaceans.ices.dk/manage/ScreenResult/1517>

I've also had an email from Lucy saying she's also had the 'Developer Mode' error today.

Thanks,
Niki
________________________________

**Comment by cmspinto** [comment]
Found the bug, but I haven't tried because I don't want to resubmit your data, could I ask you to try now?

**Comment by NikiClear** [comment]
@cmspinto 

I've tried the resubmission again this morning. The previous error has been sorted, thank you. But I've got a new error now. When I try re-upload I'm brought back to the manage dashboard with an error saying the data couldn't be uploaded (see below)

I used the metadata URL - https://gis.ices.dk/geonetwork/srv/eng/catalog.search#/metadata/04cdd959-d36b-4f6e-b3e7-622268d719b1

Any ideas what's causing this?

![image](https://user-images.githubusercontent.com/87769388/169479213-778f44e4-906b-473d-a37a-0e27179f4ac1.png)
